packages/pds/src/sequencer/sequencer.ts

```diff
@@ -222,8 +213,9 @@ export class Sequencer extends (EventEmitter as new () => SequencerEmitter) {
     return await this.sequenceEvt(evt)
   }
 
-  async sequenceHandleUpdate(did: string, handle: string): Promise<number> {
-    const evt = await formatSeqHandleUpdate(did, handle)
+  // @TODO
```

Comment:
What is the remaining TODO here (or can we nix this)?

Reply:
oops left it straggling